### DISCUSSION TITLE
AList的逻辑请教一下

### DISCUSSION THREAD

**Discussion by SlippinDylan** [AlistGo/alist#6657] [General]
我想请教一下AList的逻辑
我目前家里有一台 debian 的服务器，通过 docker 的方式部署的AList；AList中添加了阿里云盘和夸克网盘。
在群晖上，通过 cloud sync 中的 webdev 挂载AList。

那么有有个问题，如果我将文件保存到阿里云盘之后，群晖会自动下载，我想问一下，下载的时候，是不是只是占用了AList宿主机的网速和一部分存储空间用作缓存，下载完成后AList对宿主机的资源占用就基本上无了。

谢谢

**Comment by Simonzhang8** [comment]
你阿里云盘用的是302还是本地代理或代理地址？
